.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+packages
```

Comment:
Oh, yeah before we had automatic hoisting I advocated manually pulling `devDependencies` to the repo root, but now I actually think that's harmful.  Keeping the deps specified where they're actually used preserves semantic information about each package.  I'm finding the "would this still work if it weren't in an Asini repo" sniff test more and more useful.